packages/cactus-plugin-satp-hermes/src/main/typescript/core/stage-services/client/stage0-client-service.ts

```diff
@@ -381,7 +381,7 @@ export class Stage0ClientService extends SATPService {
     this.Log.info(`init-${stepTag}`);
     this.dbLogger.persistLogEntry({
       sessionID: sessionData.id,
-      type: "wrap-token",
+      type: `wrap-token-${SATPServiceType[SATPServiceType.Client]}`,
```

Comment:
Why concatenate a long string in the type to indicate whether it's the server or client, instead of using a separate variable for this distinction? Using a concatenated string will make it harder to work with these logs programmatically.